dev/tests/integration/testsuite/Magento/Customer/_files/attribute_city_store_label_address.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+//@codingStandardsIgnoreFile
+/** @var \Magento\Customer\Model\Attribute $model */
+$model = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->create(\Magento\Customer\Model\Attribute::class);
+/** @var \Magento\Store\Model\StoreManagerInterface $storeManager */
+$storeManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->create(\Magento\Store\Model\StoreManager::class);
+$model->loadByCode('customer_address', 'city');
+$storeLabels = $model->getStoreLabels();
+$websites = $storeManager->getWebsites();
```

Comment:
Please use stores instead of websites.
Currently it works correctly only because test run in single store mode.

Reply:
This does not sound right to me, I used this also in production site with `getWebsites()`?

Reply:
You could check the method **\Magento\Eav\Model\ResourceModel\Entity\Attribute::_saveStoreLabels**
which is used to save 'Store Labels' and the structure of the table which is used to store labels.
<img width="653" alt="Screen Shot 2020-03-17 at 11 52 45 AM" src="https://user-images.githubusercontent.com/9420807/76844969-c5938400-6846-11ea-9098-e9d13e75b2b6.png">
 You can see direct reference to store 'store' table.

Otherwise, they should have the name 'Website Labels', right? :)